sqlglot/dialects/teradata.py

```diff
@@ -325,3 +325,9 @@ def interval_sql(self, expression: exp.Interval) -> str:
                 return f"({multiplier} * {super().interval_sql(exp.Interval(this=expression.this, unit=exp.var('DAY')))})"
 
             return super().interval_sql(expression)
+
+        def anonymous_sql(self, expression: exp.Anonymous) -> str:
```

Comment:
you should never overwrite this, instead add Quarter as a first class function in expressions.py

Reply:
Gotcha, I'll update to do that

Reply:
I added a `exp.Quarter`, and  I tried to base this off of the places where exp.Month is used but may have missed something